diff --git a/node_modules/@opentelemetry/sdk-node/build/src/NodeSDK.js b/node_modules/@opentelemetry/sdk-node/build/src/NodeSDK.js
index a4a45f9..a3118d0 100644
--- a/node_modules/@opentelemetry/sdk-node/build/src/NodeSDK.js
+++ b/node_modules/@opentelemetry/sdk-node/build/src/NodeSDK.js
@@ -107,10 +107,6 @@
                 return new otlp_grpc_exporter_1.OTLPMetricExporter(otlpOptions);
             case 'otlp-http':
                 return new otlp_http_exporter_1.OTLPMetricExporter(otlpOptions);
-            case 'jaeger':
-                return new exporter_jaeger_1.JaegerExporter(conf);
-            case 'zipkin':
-                return new exporter_zipkin_1.ZipkinExporter(conf);
             default:
                 (_a = api_1.diag.error('unknown exporter %s, communication is not possible', name)) === null || _a === void 0 ? void 0 : _a.call(api_1.diag);
         }